changelog/unreleased/update-web-5.4.0.md

```diff
@@ -0,0 +1,9 @@
+Enhancement: Update ownCloud Web to v5.4.0-rv.3
+
+Tags: web
+
+We updated ownCloud Web to v5.4.0-rv.3. Please refer to the changelog (linked) for details on the web release.
+
+https://github.com/owncloud/web/pull/6709
+https://github.com/owncloud/ocis/pull/3437
+https://github.com/owncloud/web/releases/tag/v5.4.0-rv.3
```

Comment:
this doesn't seem right

Reply:
I followed the same convention as I found in earlier release tickets

Reply:
Also, it's a preliminary changelog item and can/will be updated towards the official web release/merge in oCIS:)

Reply:
Sorry should have pointed you directly to the typo... That's what I meant

Reply:
I'd like to merge this now (give a green CI) and then follow up with a changelog-only PR fix instead of re-running this PR though^^